Suggested text enhancements to make sure important info has better visibility (#3498)
treeverse/lakeFS: docs/why-lakefs.md

---
layout: default

> title: Why is lakeFS the data solution you’ve been missing?

Reviewer: Why is this a top level element? This looks strange:
![image](https://user-images.githubusercontent.com/3149885/173556507-f2dd6e3b-2e09-40a4-9aaa-1a8ee7563ed2.png)

Author: The "what" and the "why" is the most important things you want to know about any one topic. They should be really easy to find in a doc, either as standalone elements or inside of the same doc.

Personally I think it's important to think about snackable content, not too long articles, that's the reason why I would raise this PR, I felt the existing article was a bit compact.

I see your point on rendering but what about this approach?

lakeFS/
  Why
  What

@adipolak wdyt?

   > To learn more on how to access a specific historical commit in a repository, see our seminal post on [data reproducibility]([data reproducibility](https://lakefs.io/solving-data-reproducibility/)).

- **Establishing data quality guarantees**. The best way to deal with mistakes is to avoid them. A data source that is ingested into the lake introducing low-quality data should be blocked before exposure if possible. With lakeFS, you can achieve this by tying data quality tests to commit and merge operations via [lakeFS hooks](./setup/hooks.md).

## Additional things you should know about lakeFS

- It is format agnostic
- Your data stays in place
- It minimizes data duplication via a copy-on-write mechanism
- It maintains high performance over data lakes of any size
- It includes configurable garbage collection capabilities
- It is highly available and production ready
lakeFS integration into data lake

<img src="{{ site.baseurl }}/assets/img/lakeFS_integration.png" alt="lakeFS integration into data lake" width="60%" height="60%" />

## Start using lakeFS - available downloads

> - BINARY RELEASES. Binary packages are available for Linux/macOS/Windows on [See GitHub Releases](https://github.com/treeverse/lakeFS/releases)

Reviewer: I wouldn't use all caps anywhere

Author: > I wouldn't use all caps anywhere

you were in your h4 headers which I took away and replaced with h3 @adipolak

> - BINARY RELEASES. Binary packages are available for Linux/macOS/Windows on [See GitHub Releases](https://github.com/treeverse/lakeFS/releases)

Reviewer: ```suggestion
- BINARY RELEASES. Binary packages are available on our [GitHub Releases](https://github.com/treeverse/lakeFS/releases) page.
```

> - DOCKER IMAGES. Official Docker images are available at <https://hub.docker.com/r/treeverse/lakefs>

Reviewer: Why are docker images and binaries are part of this page? It doesn't seem related to the title.
